### PR TITLE
RDKTV-19740: Modify the frequency of CEC messages sent from TV

### PR DESCRIPTION
**Pull request by dautapankumardora**
Reason for change: Improve the CEC retry mechanism on I2c cec bus busy
Test Procedure: Refer the ticket for test procedure
Risks: Low
Priority: P1
Signed-off-by: bp-tdora114 <dautapankumar.dora@sky.uk>

### CODE REVIEW

File: HdmiCecSink/HdmiCecSink.cpp [hunk 1]
```diff
@@ -2470,7 +2477,12 @@ namespace WPEFramework
 
 					case CECDeviceParams::REQUEST_OSD_NAME :	
 					{
-						_instance->deviceList[logicalAddress].update(OSDName("NA"));
+						if ( _instance->deviceList[logicalAddress].m_isRequestRetry++ >= HDMICECSINK_REQUEST_MAX_RETRY )
```
Comment (apatel859) [inline HdmiCecSink/HdmiCecSink.cpp:2480]: So this code chanegs is adding retry to OSD name. Could we add this to VENDOR ID as well?

Comment (bijas) [reply HdmiCecSink/HdmiCecSink.cpp:2480]: Vendor id is not that mandatory like physical address or osd name.so we really need not

Comment (apatel859) [reply HdmiCecSink/HdmiCecSink.cpp:2480]: I know but providing vendor id reliably application write some logic for gaming console or something like that. So adding try there will help.

Comment (bijas) [reply HdmiCecSink/HdmiCecSink.cpp:2480]: To be failr I dont want to add it now and make the cec bus busy.As we don't know which all devices provide that information and which ones dont.When it really needed we can add.

**Review by apatel859 — APPROVED** [review]

**Review by apatel859 — APPROVED** [review]

**Pull request merged**